package.json

```diff
@@ -1,46 +1,52 @@
-    "description": "Jooby message encoders/decoders GUI",
-    "type": "module",
-    "scripts": {
-        "dev": "vite --host",
-        "build": "vite build",
-        "preview": "vite preview",
-        "lint": "eslint .",
-        "lint-cached": "eslint --cache .",
-        "lint-fix": "eslint --fix ."
-    },
-    "dependencies": {
-        "@emotion/react": "^11.11.4",
-        "@emotion/styled": "^11.11.0",
-        "@fontsource/roboto": "^5.0.12",
-        "@fontsource/roboto-mono": "^5.0.17",
-        "@mui/icons-material": "^5.15.12",
-        "@mui/lab": "^5.0.0-alpha.167",
-        "@mui/material": "^5.15.12",
-        "@types/react": "^18.2.56",
-        "@types/react-beautiful-dnd": "^13.1.8",
-        "@types/react-dom": "^18.2.19",
-        "@types/uuid": "^9.0.8",
-        "@vitejs/plugin-react": "^4.2.1",
-        "ace-builds": "^1.32.7",
-        "eslint": "^8.56.0",
-        "eslint-config-airbnb-base": "^15.0.0",
-        "eslint-plugin-custom-rules": "file:./eslint-custom-rules",
-        "eslint-plugin-import": "^2.29.1",
-        "eslint-plugin-react": "^7.33.2",
-        "eslint-plugin-react-hooks": "^4.6.0",
-        "eslint-plugin-react-refresh": "^0.4.5",
-        "jooby-codec": "^4.6.0",
-        "lz-string": "^1.5.0",
-        "react": "^18.2.0",
-        "react-beautiful-dnd": "^13.1.1",
-        "react-dom": "^18.2.0",
-        "react-json-tree": "^0.18.0",
-        "sass": "^1.71.1",
-        "uuid": "^9.0.1",
-        "vite": "^5.1.4"
-    }
+  "name": "jooby-gui",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Jooby message encoders/decoders GUI",
+  "type": "module",
+  "scripts": {
+    "dev": "vite --host",
+    "build": "vite build",
+    "preview": "vite preview",
+    "lint": "eslint .",
+    "lint-cached": "eslint --cache .",
+    "lint-fix": "eslint --fix .",
+    "test": "npx playwright test"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.0",
+    "@fontsource/roboto": "^5.0.12",
+    "@fontsource/roboto-mono": "^5.0.17",
+    "@mui/icons-material": "^5.15.12",
+    "@mui/lab": "^5.0.0-alpha.167",
+    "@mui/material": "^5.15.12",
+    "@types/react": "^18.2.56",
+    "@types/react-beautiful-dnd": "^13.1.8",
+    "@types/react-dom": "^18.2.19",
+    "@types/uuid": "^9.0.8",
+    "@vitejs/plugin-react": "^4.2.1",
+    "ace-builds": "^1.32.7",
+    "eslint": "^8.56.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-custom-rules": "file:./eslint-custom-rules",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-refresh": "^0.4.5",
+    "jooby-codec": "^4.6.0",
+    "lz-string": "^1.5.0",
+    "react": "^18.2.0",
+    "react-beautiful-dnd": "^13.1.1",
+    "react-dom": "^18.2.0",
+    "react-json-tree": "^0.18.0",
+    "sass": "^1.71.1",
+    "uuid": "^9.0.1",
+    "vite": "^5.1.4"
+  },
+  "devDependencies": {
```

Comment:
shouldn't it be in dependencies?

Reply:
no, devDependencies should contain libs and packages which need only to develop and test app and dependencies contain which needs to work app in prod

Reply:
yes

there are no prod and dev mods in this app, it's only dev

prod is serving the generated SPA elsewhere